### ISSUE TITLE
Visual Studio 2015 warnings

### ISSUE THREAD

**Issue by DJLink** [fmtlib/fmt#504]
Hello. I'm using VS 2015 with warning 4 and I'm getting several warnings.
Is it possible something on my end? I tried searching other issues but couldn't find these ones.

`fmt\printf.h(498): warning C4244: '=': conversion from 'wchar_t' to 'char', possible loss of data`

```
printf.h(415): note: while compiling class template member function 'void fmt::PrintfFormatter<char,fmt::PrintfArgFormatter<Char>>::format(fmt::BasicCStringRef<Char>)'
1>          with
1>          [
1>              Char=char
1>          ]
```

```
fmt\printf.h(523): note: see reference to function template instantiation 'void fmt::PrintfFormatter<char,fmt::PrintfArgFormatter<Char>>::format(fmt::BasicCStringRef<Char>)' being compiled
1>          with
1>          [
1>              Char=char
1>          ]
```

`\fmt\format.h(1593): note: while compiling class template member function 'void fmt::ArgVisitor<fmt::internal::ArgConverter<void>,void>::visit_ulong_long(fmt::ULongLong)`

**Comment by edorax** [comment]
There're similar warnings on VS 2012.

**Comment by foonathan** [comment]
Should be fixed by #502.
